etl/steps/data/garden/emdat/2023-09-20/natural_disasters.meta.yml

```diff
@@ -1,3 +1,9 @@
+definitions:
+  common:
+    presentation:
+      topic_tags:
+      - Natural Disasters
```

Comment:
Some indicators should probably have other tags:

- `population`: might want to use `Population Growth`
- `gdp`: might want to use `Economic Growth`

Also, do we want these indicators in Grapher?  Are they natural-disaster-specific?

Reply:
I'll tag them as "Uncategorized". They are actually removed in the grapher step.